health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/UpdateCampaign.js

```
@@ -581,48 +593,51 @@
-          }),
+              return {
+                ...prev,
+                target: [
+                  {
+                    name: `target`,
+                    error: t(`TARGET_FILE_MISSING`),
+                  },
+                ],
+              };
+            }),
             true);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)